counting.js

```diff
@@ -9,7 +9,7 @@ const error = chalk.bold.red;
 const warn = chalk.yellowBright;
 const success = chalk.green;
 var servers = {};
-var cheader = `[${vukkytils.getString("COUNTING")}]`;
+var cheader = `[${vukkytils.getString("")}]`;
```

Comment:
i think you fucked up the entire counting.js file here

Reply:
yea fucked ups a bit of an under statement, time to copy and paste it back lol